frontend/src/main/java/puf/frisbee/frontend/viewmodel/LevelViewModel.java

```diff
@@ -51,6 +53,11 @@ public void continueLevel() {
 	public int getLevel() {
 		return this.levelModel.getCurrentLevel();
 	}
+	
+	public StringProperty getLevelProperty() {
+		this.level.setValue("" + this.levelModel.getCurrentLevel());
```

Comment:
Warum braucht es da einen leeren String? Ist das so schöner als es mit der toString Funktion umzuwandeln?

Reply:
Die Konkatenation mit dem leeren Strinig bewirkt ein Casting von int nach String. Sonst kommt da ein int-Wert an, der nicht weiterverarbeitet werden kann.

Reply:
Habe das Casting mit dem leeren String durch ein schöneres Casting mit der valueOf()-Methode ersetzt. Ist so sicher besser verständlich: this.level.setValue(String.valueOf(this.levelModel.getCurrentLevel()));

Reply:
Ja das stimmt, ich frage mich nur, ob das so schöner ist als `Integer.toString(this.levelModel.getCurrentLevel())` ... aber ist wahrscheinlich Geschmackssache, es funktioniert :)